project/Build.scala

```diff
@@ -24,6 +24,7 @@ object KafkaUtilsBuild extends Build {
       "com.google.guava" % "guava" % "18.0",
       "com.quantifind" % "kafkaoffsetmonitor_2.11" % "0.4.6-SNAPSHOT",
       "io.dropwizard.metrics" % "metrics-graphite" % "3.1.2",
+      "com.signalfx.public" % "signalfx-codahale" % "0.2.1" % "provided",
```

Comment:
`provided` was added here due to file content conflicts I ran into during the build process:

```
java.lang.RuntimeException: deduplicate: different file contents found in the following:
/root/.ivy2/cache/com.signalfx.public/signalfx-codahale/jars/signalfx-codahale-0.2.1.jar:META-INF/maven/org.eclipse.jetty/jetty-io/pom.properties
/root/.ivy2/cache/org.eclipse.jetty/jetty-io/jars/jetty-io-8.1.13.v20130916.jar:META-INF/maven/org.eclipse.jetty/jetty-io/pom.properties
	at sbtassembly.Plugin$Assembly$.sbtassembly$Plugin$Assembly$$applyStrategy$1(Plugin.scala:205)
	at sbtassembly.Plugin$Assembly$$anonfun$11.apply(Plugin.scala:219)
	at sbtassembly.Plugin$Assembly$$anonfun$11.apply(Plugin.scala:217)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.immutable.HashMap$HashMap1.foreach(HashMap.scala:224)
	at scala.collection.immutable.HashMap$HashTrieMap.foreach(HashMap.scala:403)
	at scala.collection.immutable.HashMap$HashTrieMap.foreach(HashMap.scala:403)
	at scala.collection.immutable.HashMap$HashTrieMap.foreach(HashMap.scala:403)
	at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:251)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:105)
	at sbtassembly.Plugin$Assembly$.applyStrategies(Plugin.scala:221)
	at sbtassembly.Plugin$Assembly$.apply(Plugin.scala:171)
	at sbtassembly.Plugin$.sbtassembly$Plugin$$assemblyTask(Plugin.scala:157)
	at sbtassembly.Plugin$$anonfun$baseAssemblySettings$6.apply(Plugin.scala:366)
	at sbtassembly.Plugin$$anonfun$baseAssemblySettings$6.apply(Plugin.scala:365)
	at scala.Function10$$anonfun$tupled$1.apply(Function10.scala:35)
	at scala.Function10$$anonfun$tupled$1.apply(Function10.scala:34)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[error] (*:assembly) deduplicate: different file contents found in the following:
[error] /root/.ivy2/cache/com.signalfx.public/signalfx-codahale/jars/signalfx-codahale-0.2.1.jar:META-INF/maven/org.eclipse.jetty/jetty-io/pom.properties
[error] /root/.ivy2/cache/org.eclipse.jetty/jetty-io/jars/jetty-io-8.1.13.v20130916.jar:META-INF/maven/org.eclipse.jetty/jetty-io/pom.properties
[error] Total time: 805 s, completed Nov 25, 2019 7:47:43 PM
```